GCTC-NTGC/gc-digital-talent
# Admin - Remove action links in Table component

Resolves #3823.

## Summary
Removes action links in the client-controlled Table component that are not used.

## Testing
1. Build admin app `npm run production --workspaces=admin`
2. Navigate to `/en/admin/dashboard`
3. Confirm action links in table footer are no longer present
